File: 백준/Bronze/2577. 숫자의 개수/숫자의 개수.py
import sys

# 입력 받기
A = int(sys.stdin.readline().strip())
B = int(sys.stdin.readline().strip())
C = int(sys.stdin.readline().strip())

# 곱셈 수행
result = A * B * C

result_str = str(result)

# 숫자 빈도 카운트
digit_count = {}
for i in range(10):
    digit_count[str(i)] = 0

    
for char in result_str:
    digit_count[char] += 1

# 결과 출력
for i in range(10):
    print(digit_count[str(i)])

File: 백준/Bronze/2577. 숫자의 개수/숫자의 개수.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n').map(Number);

const [A, B, C] = input;
const result = A * B * C;
const resultStr = result.toString();

const digitCount = {};
for (let i = 0; i < 10; i++) {
    digitCount[i] = 0;
}

for (const char of resultStr) {
    digitCount[char]++;
}

// 결과 출력
for (let i = 0; i < 10; i++) {
    console.log(digitCount[i]);
}
